src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -168,6 +169,7 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     decomp->add_matcher<ngraph::pass::DropoutWithRandomUniformReplacer>();
     decomp->add_matcher<ngraph::pass::TransposeReshapeEliminationForMatmul>();
     decomp->add_matcher<ov::pass::EyeDecomposition>();
+    decomp->add_matcher<ov::pass::UniqueDecomposition>();
```

Comment:
Does any plugin support Unique op?
Should we disable this transformation via pass_config in cpu transformation pipeline?

Reply:
No plugins yet. We have a chance to have CPU to support it in this release but it is not exact information. So I make this transformation by default